src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -55,10 +55,18 @@ IOptions<DependencyUpdateErrorProcessorOptions> options
             _logger = logger;
         }
 
-       [CronSchedule("0 0 0/1 1/1 * ? *", TimeZones.PST)]
-       public async Task ProcessDependencyUpdateErrorsAsync()
+        [CronSchedule("0 0 0/1 1/1 * ? *", TimeZones.PST)]
+        public async Task ProcessDependencyUpdateErrorsAsync()
         {
-            if (_options.IsEnabled)
```

Comment:
We probably shouldn't undo your previous checkin. :-)

Reply:
Yes, my VS was acting weird and github was down. I have fixed that locally, will check in my changes tomorrow :)